src/workerd/jsg/jsg.h

```diff
@@ -1970,6 +1971,14 @@ class Lock {
   using Logger = void(Lock&, kj::StringPtr);
   void setLoggerCallback(kj::Function<Logger>&& logger);
 
+  // ---------------------------------------------------------------------------
+  // Misc. Stuff
+
+  void requestGcForTesting() const;
+  // Sends an Immediate request for full GC, this function is to ONLY be used in testing, otherwise
```

Comment:
nit; There's "otherwise" twice.

Also, is "Immediate" capitalized for some reason?

Reply:
Good catch, the Immediate was muscle memory from a past project btw.

Reply:
Done